testing/adios2/engine/inline/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
+
+add_executable(TestInlineWriteRead TestInlineWriteRead.cpp)
+target_link_libraries(TestInlineWriteRead adios2 gtest)
+
+if(ADIOS2_HAVE_MPI)
+
+  target_link_libraries(TestInlineWriteRead MPI::MPI_C)
+
```

Comment:
add here `set(extra_test_args EXEC_WRAPPER ${MPIEXEC_COMMAND})`  before any call to `gtest_add_tests`

Reply:
Thanks! I'll try it